experiment_buddy/experiment_buddy.py

```diff
@@ -219,11 +223,15 @@ def _ask_experiment_id(cluster, sweep):
         experiment_id = tkinter.simpledialog.askstring(title, "experiment_id")
         root.destroy()
     except:
-        experiment_id = input(f"Running on {title} \ndescribe your experiment (experiment_id):\n")
+        if os.environ['BRANCH_TESTING']:
```

Comment:
what is BRANCH_TESTING?

Reply:
as in, add a description somewhere

Reply:
if the purpose is to avoid the exp_id on testing we could be using something like $BUDDY_SKIP_EXPERIMENTID or $BUDDY_EXPERIMENTID

Reply:
I needed a way to quickly test the developing branch on the cluster, using the name of the branch with pip is a way.
To note that this change has to be done also in the requirements.txt of the example project

Reply:
Put buddy in front of the buddy variables so we don't risk to conflict with others

Reply:
Done